utils/deploy-aws/ansible/roles/node/templates/redis_conf.j2

```diff
@@ -7,4 +7,4 @@ supervised auto
 port {{ item['port'] }}
 protected-mode no
 dbfilename redis.rdb
-loadmodule {{ install_dir }}/lib/redisraft.so addr {{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}:{{ item['port'] }} slot-config {{ item['slot-config'] }} sharding yes {% for k, v in redisraft_config.items() -%} {{ k }} {{ v }} {% endfor %}
+loadmodule {{ install_dir }}/lib/redisraft.so raft.addr {{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}:{{ item['port'] }} raft.slot-config {{ item['slot-config'] }} raft.sharding yes {% for k, v in redisraft_config.items() -%} {{ k }} {{ v }} {% endfor %}
```

Comment:
```suggestion
loadmodule {{ install_dir }}/lib/redisraft.so
raft.addr {{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}:{{ item['port'] }}
raft.slot-config {{ item['slot-config'] }}
raft.sharding yes
{% for k, v in redisraft_config.items() -%}
{{ k }} {{ v }}
{% endfor %}
```

Since these are now first-class config arguments, we can have them as part of the config file.

Reply:
Ah, nice. I didn't realize this usage.